src/jit/x64/emit.dasc

```diff
@@ -1917,9 +1917,38 @@ static void load_call_arg(MVMThreadContext *tc, MVMJitCompiler *compiler, MVMJit
         | mov TMP6, qword WORK[arg.v.reg];
         | lea TMP6, STOOGE:TMP6->data;
         break;
+    case MVM_JIT_REG_DYNIDX:
+        | get_cur_op TMP5;
+        | xor TMP6, TMP6;
+        | mov TMP6w, word [TMP5 + arg.v.reg*2]
+        | mov TMP6, qword [WORK + TMP6*8];
+        break;
     case MVM_JIT_DATA_LABEL:
         | lea TMP6, [=>(arg.v.lit_i64)];
         break;
+    case MVM_JIT_SAVED_RV:
+        | mov TMP6, qword [rbp-0x28];
+        break;
+    case MVM_JIT_ARG_I64:
+        | mov TMP6, TC->cur_frame;
+        | mov TMP6, FRAME:TMP6->args;
+        | mov TMP6, qword REGISTER:TMP6[arg.v.lit_i64];
+        break;
+    case MVM_JIT_ARG_PTR:
+        | mov TMP6, TC->cur_frame;
+        | mov TMP6, FRAME:TMP6->args;
+        | mov TMP6, qword REGISTER:TMP6[arg.v.lit_i64];
+        | mov TMP6, aword STOOGE:TMP6->data;
+        break;
```

Comment:
Same as above, but more importantly, I think this should have a more descriptive name because it *dereferences* the data pointer of a MVMObjectStooge, which I wouldn't expect from the name.

Reply:
Naming is hard :) The enums are named after what is passed. Pointers are passed as CPointer objects which contain the actual pointer in their data. So it's indeed one level of dereferencing, but the result is still a pointer which is passed to the native function. So MVM_JIT_ARG_PTR does indeed result in a Pointer being passed. It just does the unboxing, too.
Maybe a comment like this ^^^ will suffice?

Reply:
Yep, I'll be perfectly happy with a comment.